src/components/Tree/TreeItem.tsx

```diff
@@ -126,162 +123,165 @@ export const TreeItem = ({
-                    }
-                    break;
+                    case 'ArrowDown':
+                        event.preventDefault();
+                        if (treeItemState) {
+                            const nextItemToFocus = getNextItemToFocus(id, treeItemState, treeState);
+                            nextItemToFocus?.domElement?.focus();
+                        }
+                        break;
+                }
             }
         },
         [id, onExpand, onSelect, treeItemState, treeState],
     );
 
-    const childrenArray = useMemo(
-        () =>
-            children
-                ? cloneThroughFragments(children, {
-                      parentId: id,
-                      level: (level || 0) + 1,
-                  })
-                : [],
+    const enhancedChildren = useMemo(
+        () => (children ? cloneThroughFragments(children, { parentId: id, level: (level || 0) + 1 }) : []),
         [children, id, level],
     );
 
-    const { draggableEnhancedChildren } = useDraggableEnhancedChildren({
-        accept: getAcceptTypes(accepts ?? treeId, 'before'),
-        onDrop: handleDrop,
-        children: childrenArray,
-        sort,
-    });
-    const enhancedChildren: ReactNode = draggable ? draggableEnhancedChildren : childrenArray;
-
     const hasChildren = Children.count(enhancedChildren) > 0;
 
     return (
-        <li data-test-id="tree-item" ref={drag} style={{ opacity }}>
-            <DropZone
-                data-position={DropZonePosition.Within}
-                data={{
-                    targetItem: { id, sort, type },
-                    position: DropZonePosition.Within,
-                }}
-                onDrop={handleDrop}
-                accept={getAcceptTypes(accepts ?? treeId, 'within')}
-            >
-                <div
-                    ref={itemRef}
-                    className={merge([
-                        'tw-flex tw-py-2 tw-px-2.5 tw-no-underline tw-leading-5 tw-h-10',
-                        FOCUS_VISIBLE_STYLE,
-                        treeState.selectedIds.has(id)
-                            ? 'tw-font-medium tw-bg-box-selected-strong tw-text-box-selected-strong-inverse hover:tw-bg-box-selected-strong-hover hover:tw-text-box-selected-strong-inverse-hover'
-                            : 'tw-text-text hover:tw-bg-box-neutral-hover hover:tw-text-box-neutral-inverse-hover',
-                    ])}
-                    role="treeitem"
-                    aria-label={label}
-                    aria-level={treeItemState?.level}
-                    aria-selected={treeState.selectedIds.has(id)}
-                    aria-expanded={treeState.expandedIds.has(id)}
-                    tabIndex={0}
-                    onClick={handleSelect}
-                    onKeyDown={handleKeyDown}
-                    onMouseEnter={handleMouseEnter}
-                    onMouseLeave={handleMouseLeave}
+        <>
+            {(propsDraggable ?? draggable) && sort === 0 ? (
```

Comment:
what is propsDraggable?

Reply:
`Add draggable on a per item basis.`
Allows remove dragging on one specific item (cover page and brandlink shouldn't be draggable for example)

Reply:
I would give it a more meaningful name then, as it is confusing in my opinion. But that is not urgent ATM anyway. 👍